terraform/deployments/chat/redis.tf

```
@@ -0,0 +1,44 @@
+locals {
+  chat_redis_name     = "chat-redis"
+  elasticache_subnets = data.terraform_remote_state.infra_networking.outputs.private_subnet_elasticache_ids
+}
+
+resource "aws_elasticache_subnet_group" "chat_redis_cluster" {
+  name       = local.chat_redis_name
+  subnet_ids = local.elasticache_subnets
+}
+
+resource "aws_security_group" "chat_redis_cluster" {
+  name        = local.chat_redis_name
+  vpc_id      = data.tfe_outputs.vpc.nonsensitive_values.id
+  description = "GOV.UK Chat Redis cluster"
+  tags = {
+    Name = local.chat_redis_name
+  }
+}
+
+resource "aws_elasticache_replication_group" "chat_redis_cluster" {
+  apply_immediately          = var.chat_redis_cluster_apply_immediately
+  replication_group_id       = local.chat_redis_name
+  description                = "Redis for Sidekiq queues"
```

Just another sidekiq reference to sort out
```suggestion
  description                = "Redis for GOV.UK Chat"
```